packages/typespec-azure-portal-core/lib/decorators.tsp

```diff
@@ -0,0 +1,66 @@
+import "../dist/src/index.js";
+
+using TypeSpec.Reflection;
+namespace Azure.Portal;
+
+/**
+ * Provides a Model describing about of ARM resource.
+ * @param options Property options allows more detailed infomation about the resourceType.
+ */
+extern dec about(target: Model, options: AboutOptions);
+
+/**
+ * Provides a Model customizing browse view of ARM resource.
+ * @param options Property options provides browsing information of the resourceType.
+ */
+extern dec browse(target: Model, options: BrowseOptions);
+
+/**
+ * Provides a Model marketplace offer information of ARM resource.
+ * @param options Property options provides marketplace offer information of the resourceType.
+ */
+extern dec marketplaceOffer(target: Model, options: marketplaceOfferOptions);
+
+/**
+ * Provides a Model Property a display name
+ */
+extern dec displayName(target: ModelProperty, name: valueof string);
+
+//extern dec essentials(target: ModelProperty);
```

Comment:
is this supposed to be commented out(same below)

Reply:
yes :) keeping it for next iteration! :) is that okay?

Reply:
can you just add a note above it then, just in case this never get to get added for some reason we know why it is here commented out

Reply:
yes just did! thank you!